assets/src/application/ui/display/ItemCount/index.tsx

```diff
@@ -7,14 +7,22 @@ import { getPropsAreEqual } from '@appServices/utilities';
 import './style.scss';
 
 interface ItemCountProps extends BadgeProps {
-	zeroCountChar?: string | JSX.Element;
 	children: React.ReactNode;
+	emphasizeZero?: boolean;
+	zeroCountChar?: string | JSX.Element;
 }
 
-const ItemCount: React.FC<ItemCountProps> = ({ children, count, title = ' ', zeroCountChar = '!', ...props }) => {
+const ItemCount: React.FC<ItemCountProps> = ({
+	children,
+	count,
+	emphasizeZero = true,
+	title = ' ',
+	zeroCountChar = '!',
+	...props
+}) => {
 	const className = classNames(props.className, 'ee-item-count', {
 		'ee-item-count--has-items': count > 0,
-		'ee-item-count--no-items': count === 0,
+		'ee-item-count--no-items': count === 0 && emphasizeZero,
```

Comment:
just to help me understand, what Is the purpose behind emphasizing zero?

Reply:
See the screenshot I have attached